### DISCUSSION TITLE
Dark theme in released version

### DISCUSSION THREAD

**Discussion by kkarbowiak** [doublecmd/doublecmd#656] [General]
Due to answers in #649 I know that dark mode is available in snapshot builds.

Is it going to be available in the next release? If not, what is blocking it?

**Comment by alexx2000** [comment]
It will be in the next major release 1.1.0.

> what is blocking it?

#462
#440

**Reply by kkarbowiak** [reply]
Thank you.

**Comment by kkarbowiak** [comment]
Dark theme is shipped so I am closing this. Thanks!
